Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.py

```diff
@@ -0,0 +1,437 @@
+                    ):
+                        passwords_section.append(
+                            f"{exposed_secret['details']['clear_text_hint']} (__{exposed_secret['type']}__)"
+                        )
+                    if exposed_secret["hashes"]:
+                        for hash_password in exposed_secret["hashes"]:
+                            # When user do not have permissions for domain
+                            # we will receive hash_prefix instead of full hash.
+                            hash_value = hash_password.get(
+                                "hash"
+                            ) or hash_password.get("hash_prefix")
+                            hash_algorithm = hash_password["algorithm"]
+                            passwords_section.append(
+                                f"{hash_value} (__{hash_algorithm}__)"
+                            )
+                    if credential.get("authorization_service"):
+                        passwords_section.append(
+                            f"Authorization service url: {credential['authorization_service']['url']}\n"
+                        )
+                    if credential.get("exfiltration_date"):
+                        exfiltration_date = datetime.strptime(
+                            credential["exfiltration_date"], ISO_DATE_FORMAT
+                        ).strftime("%b %Y")
+                        passwords_section.append(
+                            f"Exfiltration date: {exfiltration_date}"
+                        )
+                    for dump in credential["dumps"]:
+                        dump_downloaded = dump.get("downloaded", "")
+                        if dump_downloaded:
+                            dump_downloaded = datetime.strptime(
+                                dump_downloaded, ISO_DATE_FORMAT
+                            ).strftime("%b %Y")
+                            dump_text = f"__{dump['name']}__, {dump_downloaded},  {password_number_text}"
+                        else:
+                            dump_text = (
+                                f"__{dump['name']}__, {password_number_text}"
+                            )
+                        # There might be the same dumps in the response so we show only one.
+                        if dump_text not in dumps_section:
+                            dumps_section.append(dump_text)
+                            if dump.get("type"):
+                                dumps_section.append(
+                                    f"__Dump type:__ {dump['type']}"
+                                )
+                            dumps_section.append(
+                                f"{dump.get('description', '')}"
+                            )
+                        for breach in dump.get("breaches", []):
+                            breach_date = breach.get("breached")
+                            if breach_date:
+                                breach_date = datetime.strptime(
```

Comment:
please use `dateparser.parse` instead of `strptime`

Reply:
Cool, didn't know about that lib